### ISSUE TITLE
ULMFiT pretraining

### ISSUE THREAD

**Issue by yunzhe-tao** [fastai/fastai#674]
Hi,

I am wondering what perplexity you got for the pre-trained model on WikiText-103, which was not mentioned in your paper. I am asking this because I tried your pre-training code with setting QRNN=True. After 12 epochs training, I can only get the model with ppl > 250 (val loss=5.5). Is it close to the provided pre-trained model?

Another question is that I saw the example command of running pre-training is on German version of WikiText. Is there any reason to do this? Or there is no performance difference between German and English.

Thank you!

**Comment by sgugger** [comment]
Hi there,

This would be best asked on the [forum](http://forums.fast.ai/). The best way to answer your question is probably to load the model and validate it.
